components/common/layout.tsx

```diff
@@ -3,11 +3,11 @@ import Head from 'next/head';
 import Navbar from './navbar/navbar';
 
 interface LayoutProps {
-    pageName: string;
+    pageTitle: string;
```

Comment:
Renamed prop to be a bit more explicit to indicate this is the title shown in the browser tab.